src/api/entities/__tests__/NumberedPortfolio.ts

```diff
@@ -89,20 +90,42 @@ describe('NumberedPortfolio class', () => {
   });
 
   describe('method: getName', () => {
+    const id = new BigNumber(1);
+    const did = 'someDid';
+    const portfolioName = 'someName';
     test('should return the name of the Portfolio', async () => {
-      const id = new BigNumber(1);
-      const did = 'someDid';
-      const portfolioName = 'someName';
-      const rawPortfolioName = dsMockUtils.createMockText(portfolioName);
       const numberedPortfolio = new NumberedPortfolio({ id, did }, context);
-
+      const spy = jest.spyOn(numberedPortfolio, 'exists').mockResolvedValue(true);
```

Comment:
Looks like we don't use jest.spyOn, but its already installed, and this seems to be a good use case for it vs setting up the boiler plate thats in the exists check.